static/bidder-info/adport.yaml

```diff
@@ -2,8 +2,8 @@ endpoint: "https://adport.pbsadverxo.com/auction?id={{.AdUnit}}&auth={{.TokenID}
 aliasOf: "adverxo"
 userSync:
   iframe:
-    url: https://adport.pbsadverxo.com/usync?type=iframe&gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
+    url: https://cittamatra.com/usync?type=iframe&gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
     userMacro: $UID
   redirect:
```

Comment:
Verified bidsmind and adport redirect urls redirect to setuid endpoint with uid in cookie
<img width="1250" alt="Screenshot 2025-06-05 at 3 20 15 PM" src="https://github.com/user-attachments/assets/91a02e89-ede2-4bc8-8bf6-794fe29d91e1" />